torch/nn/modules/linear.py

```diff
@@ -178,4 +179,78 @@ def extra_repr(self) -> str:
             self.in1_features, self.in2_features, self.out_features, self.bias is not None
         )
 
+
+class LazyLinear(LazyModuleMixin, Linear):
+    r"""A :class:`torch.nn.Linear` module with lazy initialization.
+
+    This module will initialize the `weight` and `bias` parameters after
+    the first call to ``forward`` is done. After that the module will
+    become a :class:`torch.nn.Linear` dropping all the lazy initialization
+    logic.
+
+    Be sure to run "dummy" ``forward`` once to initialize all parameters in case
```

Comment:
I think this should reference the lazy documentation to avoid redundancy and stay up to date.

Reply:
Per above comment, this sentence can be removed.